pkg/alertmanager/multitenant.go

```diff
@@ -697,34 +704,33 @@ func (am *MultitenantAlertmanager) stopping(_ error) error {
 }
 
 // loadAlertmanagerConfigs Loads (and filters) the alertmanagers configuration from object storage, taking into consideration the sharding strategy.
-func (am *MultitenantAlertmanager) loadAlertmanagerConfigs(ctx context.Context) (map[string]alertspb.AlertConfigDesc, error) {
+// Returns the list of all users with a configuration, and the set of users which should be configured in this instance.
```

Comment:
> and the set of users which should be configured in this instance

It returns all discovered users, not the ones that should be configured in this instance.

Reply:
I actually misread the comment initially. It's good.